src/main/resources/static/styles.css

```diff
@@ -0,0 +1,60 @@
+
```

Comment:
Not sure if intended or not with the styles but I would check the margins in the mobile view
![Screenshot from 2024-11-04 13-42-04](https://github.com/user-attachments/assets/66c0861b-6a75-400b-a2fc-afd19ea2ae08)
![Screenshot from 2024-11-04 13-41-18](https://github.com/user-attachments/assets/0efa6e7e-9325-4ec1-b7f2-8147093adf10)

Reply:
![image](https://github.com/user-attachments/assets/7734af5a-996d-428c-b49a-43014f008290)
changed

Reply:
Great! I liked the idea of the transparency!
As a personal preference I will darken it a little still so is more readable